.travis.yml

```diff
@@ -45,18 +54,21 @@ before_install: |
     pyenv install $PYTHON 
     pyenv virtualenv $PYTHON MYVERSION
     source ~/.pyenv/versions/MYVERSION/bin/activate
+    pyenv global $PYTHON
     python --version 
     python -c 'import sys;print(sys.version)' 
     python -c 'import sys;print(sys.executable)' 
   fi
 install:
 # we have this here so we can see where python is installed and hardcode in our tests
 # else when running npm test, the python version picked is completely different :(
+# python-dev is to ensure jupyter installs properly
+  - sudo apt-get install python-dev
   - python --version 
   - python -c 'import sys;print(sys.executable)'
   - pip install -r requirements.txt
```

Comment:
You can also use `python -m pip` to guarantee what version of Python you're using.

Reply:
how do i do that?